src/aere_repl.hrl

```diff
@@ -13,10 +13,13 @@
 -type chain_state() :: {ready, aefa_chain_api:state()}
                      | {breakpoint, aefa_engine_state:state()}.
 
+-type contract_state() :: {aeso_syntax:type(), aeso_syntax:expr(), aeb_fate_data:fate_type()}.
```

Comment:
Keeping expr is unnecessary

Reply:
Removed here 73959060d33c75872230ba803be08f0d0e2da939